python/decrypt.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
```

Comment:
how about making a folder for bash scripts? it starts with 2 but eventually you have a couple more and it starts cluttering the repo